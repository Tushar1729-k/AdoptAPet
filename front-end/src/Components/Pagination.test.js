import React from 'react';
import Paginate from "./Pagination";
import renderer from 'react-test-renderer';

const paginate = (num) => {
    return
}

test("Number of Pages Properly Rendered", () => {
    const component = renderer.create(
        <Paginate totalItems={600} itemsPerPage={100} paginate={paginate}/>,
      );
      let tree = component.toJSON();
      expect(tree).toMatchSnapshot();
})
