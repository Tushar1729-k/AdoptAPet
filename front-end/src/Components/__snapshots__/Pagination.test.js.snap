// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Number of Pages Properly Rendered 1`] = `
<div>
  <ul
    className="pagination"
  >
    <li
      className="page-item"
    >
      <a
        className="page-link"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          aria-hidden="true"
        >
          ‹
        </span>
        <span
          className="visually-hidden"
        >
          Previous
        </span>
      </a>
    </li>
    <li
      className="page-item active"
    >
      <span
        className="page-link"
        disabled={false}
        onClick={[Function]}
      >
        1
        <span
          className="visually-hidden"
        >
          (current)
        </span>
      </span>
    </li>
    <li
      className="page-item"
    >
      <a
        className="page-link"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex={0}
      >
        2
      </a>
    </li>
    <li
      className="page-item"
    >
      <a
        className="page-link"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex={0}
      >
        3
      </a>
    </li>
    <li
      className="page-item"
    >
      <a
        className="page-link"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex={0}
      >
        4
      </a>
    </li>
    <li
      className="page-item"
    >
      <a
        className="page-link"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex={0}
      >
        5
      </a>
    </li>
    <li
      className="page-item"
    >
      <a
        className="page-link"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex={0}
      >
        6
      </a>
    </li>
    <li
      className="page-item"
    >
      <a
        className="page-link"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          aria-hidden="true"
        >
          …
        </span>
        <span
          className="visually-hidden"
        >
          More
        </span>
      </a>
    </li>
    <li
      className="page-item"
    >
      <a
        className="page-link"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          aria-hidden="true"
        >
          ›
        </span>
        <span
          className="visually-hidden"
        >
          Next
        </span>
      </a>
    </li>
  </ul>
</div>
`;
